README.md

```diff
@@ -122,14 +122,14 @@ $ slather coverage -x --output-directory path/to/xml_report
 ### Coverage for code included via CocoaPods
 
 If you're trying to compute the coverage of code that has been included via
-CocoaPods, you will need to tell CocoaPods to use the slather plugin by
+CocoaPods, you will need to tell CocoaPods to use the Slather plugin by
```

Comment:
Yes, but in this case it's being used in a sentence as a proper noun, so it's capitalized. The sample code is correctly lowercase.